doc/js_tutorials/js_assets/js_image_classification.html

```diff
@@ -0,0 +1,338 @@
+                        <tr>
+                                <th scope="row">2</th>
+                                <td id="label1" align="center"></td>
+                                <td id="prob1" align="center"></td>
+                        </tr>
+                        <tr>
+                                <th scope="row">3</th>
+                                <td id="label2" align="center"></td>
+                                <td id="prob2" align="center"></td>
+                        </tr>
+                    </tbody>
+                </table>
+                <p id='time' align="left"></p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <div class="caption">
+                    canvasInput <input type="file" id="fileInput" name="file" accept="image/*">
+                </div>
+            </td>
+            <td></td>
+        </tr>
+    </table>
+</div>
+
+<div>
+    <h3>Help function</h3>
+    <p>1.Main loop in which will read the image from canvas and do inference. You can modify the parameters to run different models.</p>
+    <textarea class="code" rows="30" cols="100" id="codeEditor" spellcheck="false"></textarea>
+    <p>2.Load labels from txt file and process it into an array.</p>
+    <textarea class="code" rows="30" cols="100" id="codeEditor1" spellcheck="false"></textarea>
+    <p>3.Get blob from image as input for net, and standardize it with <b>mean</b> and <b>std</b>.</p>
+    <textarea class="code" rows="10" cols="100" id="codeEditor2" spellcheck="false"></textarea>
+    <p>4.Fetch model from network and save to emscripten file system.</p>
+    <textarea class="code" rows="20" cols="100" id="codeEditor3" spellcheck="false"></textarea>
+    <p>5.Run the model to get the output vector and record the inference time.</p>
+    <textarea class="code" rows="25" cols="100" id="codeEditor4" spellcheck="false"></textarea>
+    <p>6.The post-processing, including softmax if needed and get the top classes from the output vector.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor5" spellcheck="false"></textarea>
+    <p class="err" id="errorMessage"></p>
+</div>
+
+<script src="utils.js" type="text/javascript"></script>
+
+<script id="codeSnippet" type="text/code-snippet">
+inputSize = [224,224];
+mean = [104, 117, 123];
+std = 1;
+rgb = true;
+labels = labels1000;
+needSoftmax = false;
+
+input = getBlodFromImage(inputSize, mean, std, rgb);
+modelPath = "bvlc_googlenet.caffemodel";
+modelUrl = "https://raw.githubusercontent.com/akineeic/opencv_model/master/image_classification/caffe/bvlc_googlenet.caffemodel";
+configPath = "bvlc_googlenet.prototxt";
+configUrl = "https://raw.githubusercontent.com/akineeic/opencv_model/master/image_classification/caffe/bvlc_googlenet.prototxt";
+modelInfo = {
+    modelPath: modelPath,
+    modelUrl: modelUrl,
+    configPath: configPath,
+    configUrl: configUrl
+}
+loadModel(() => {
+    runModel();
+});        
```

Comment:
Please use the `await` to make it easy to read.

I think the main flow should be something like this:
```js
const path = await loadModel(modelInfo);
const labels = await loadLables(labelsUrl);
const input = getBlobFromImage(imageElementId, preprocessInfo);
const net = cv.readNet(path);
net.setInput(input);
const result = net.forward();
const prob = softmax(result)
const classes = getTopClasses(prob, labels, topK)
```